sdcm/sct_events/filters.py

```diff
@@ -25,13 +25,14 @@ def __init__(self,
                  db_event: Union[LogEventProtocol, Type[LogEventProtocol]],
                  line: Optional[str] = None,
                  node: Optional = None,
-                 extra_time_to_expiration: Optional[int] = 0):
+                 extra_time_to_expiration: Optional[int] = 0,
+                 regex: Optional[str] = None):
```

Comment:
I would suggest following the implementation in `EventsFilter`
```suggestion
                 regex: Optional[Union[str, re.Pattern]]= None):
```